index.css

```diff
@@ -154,6 +153,7 @@
     display: flex;
     flex-direction: column;
     margin-bottom: 1rem;
+    word-break: break-word;
```

Comment:
Oooo missed this, probably not needed anymore?

Reply:
Just checked. Still necessary for proper wrapping in nav! I'll resolve and merge :)